src/log/file/log_file_daemon.cc

```diff
@@ -8,6 +8,8 @@
 
 #include "squid.h"
 
+#include "compat/socket.h"
```

Comment:
API provided by this include is not used in this .cc file.
```suggestion
```

Reply:
removed